Python/optimizer_cases.c.h

```diff
@@ -2016,3 +1980,20 @@
             break;
         }
 
+        case _DEOPT: {
+            break;
+        }
+
+        case _SIDE_EXIT: {
+            break;
+        }
+
+        case _ERROR_0: {
+            break;
+        }
+
+        case _ERROR_N: {
+            stack_pointer += -oparg;
+            break;
+        }
```

Comment:
You know, I never really considered that we could run the optimizer on side exits (especially once they have reconstruction code, etc.). That's pretty cool.

Reply:
The optimizer will have exited before we see any of these, but we could optimize side exits if we wanted.